packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.stories.tsx

```diff
@@ -1,26 +1,36 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { MockApp } from '../../../.storybook/components/MockApp';
-import PaymentLegalBlub from './index';
```

Comment:
Now I'll never be able to think of it as anything other than Legal Blubs. 😢👍

Reply:
😂 Ditto. When I updated the file, it took me a hot minute to realize the misspelling. RIP `PaymentLegalBlub` 😢 .